### PR TITLE
Lit tests for target_compile_features and file command CMake syntax

### PR DESCRIPTION
**Pull request by TejaX-Alaghari**
Added lit tests for the following 2 CMake syntax -

1. target_compile_features
2. file

### CODE REVIEW

File: clang/test/dpct/cmake_migation/case_011/input.cmake [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+target_compile_features(cuexe PRIVATE cxx_std_20)
+target_compile_features(cuexe 
+  PUBLIC 
+  cuda_std_20)
+
+TARGET_COMPILE_FEATURES(cuexe PUBLIC c_std_23)
+target_compile_features(cuexe PRIVATE cxx_std_23)
+target_compile_features(cuexe 
+  PUBLIC 
+  cuda_std_23)
```
Comment (tomflinda) [inline clang/test/dpct/cmake_migation/case_011/input.cmake]: For this part, you can commented as a TODO here, as it should be untouched, just like https://github.com/oneapi-src/SYCLomatic/blob/8eafeeafbdc6a04021db4acb32351fc6904b8ed2/clang/test/dpct/cmake_migation/case_021/input.cmake#L7C3-L7C4

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/test/dpct/cmake_migation/case_011/input.cmake (hunk 1 above)
Comment (TejaX-Alaghari) [reply clang/test/dpct/cmake_migation/case_011/input.cmake]: Added comments as suggested

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**